site/scripts/prepare-blocks.ts

```diff
@@ -489,14 +495,42 @@ const script = async () => {
       });
 
       const blockMetadata = await fs.readJson(blockMetadataPath);
-      blockMetadata.unstable_hubInfo = {
-        ...hubInfo,
-        checksum: blockInfoChecksum,
-        commit: blockInfo.commit,
-        preparedAt: new Date().toISOString(),
-        name: blockInfo.name,
+
```

Comment:
Instead of a 2-stage process where we add some stuff to the metadata files at `/public/blocks`, which are served via URI, and then add some _more_ metadata to `blocks-data.json`, we just add all the extra metadata to the files in the first place here.